webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2936,15 +2934,32 @@ var userOverride = {
                             'required',
                             preLeadCollection.required
                         );
-                    kmChatInput.setAttribute(
-                        'placeholder',
-                        preLeadCollection.placeholder || ''
-                    );
                     kmChatInput.setAttribute('class', preLeadCollectionClass);
-                    kmChatInput.setAttribute(
-                        'aria-label',
-                        preLeadCollection.field
-                    );
+                    if (
+                        preLeadCollection.element == 'select' &&
+                        preLeadCollection.options &&
+                        mckMessageService.checkArray(preLeadCollection.options)
+                    ) {
+                        var dropDownOptions = preLeadCollection.options;
+                        var setDropDown = _this.createSelectFieldDropdown(
```

Comment:
dropDownOptionsdropDownOptions and setDropDown variable is not required, as it is used only once, directly use 
kmChatInput = _this.createSelectFieldDropdown(preLeadCollection.options, kmChatInput);